components/example/ShowCase.vue

```diff
@@ -22,7 +22,7 @@ const name = ref<string>(props.nameInitialValue)
   <div class="p-1">
     <h1>Form:</h1>
     <p>An ant design form that emits a submit on click to the button. Note that we use is via an "auto-import", we did not need to perform a manual import in the scripts section.</p>
-    <form @submit.prevent="emits('submit', { name })">
+    <form @submit.prevent="() => emits('submit', { name })">
```

Comment:
Just for insights, why are we changing it now? Did it not work before and we shipped it or is this just a better practice?

Reply:
The types were updated via the `vue-tsc` package bump -> this + the change to the `emits` definition in the `script setup` part above reflects the need to statisy a signature like: `(): undefined` (:

Reply:
Amazing! Thanks for the explanation ❤️